bots/store.js

```diff
@@ -0,0 +1,52 @@
+const { createSelector } = require('redux-bundler')
+const { merge } = require('ramda')
+
+module.exports = {
+  name: 'bots',
+  getReducer: function () {
+    const initialState = {
+      data: null,
+      isLoading: false,
+      error: null
+    }
+
+    return (state = initialState, { type, bot, error }) => {
+      if (type === 'FIND_BOTS_STARTED') {
+        return merge(state, { isLoading: true })
+      } else if (type === 'FIND_BOTS_FINISHED') {
+        return merge(state, {
+          data: merge(state.data, { [bot.botId]: bot }),
+          error: null,
+          isLoading: false
+        })
+      } else if (type === 'FIND_BOTS_FAILED') {
+        return merge(state, {
+          error,
+          isLoading: false
+        })
+      }
+      return state
+    }
+  },
+  doFindBots: data => ({ dispatch, client }) => {
```

Comment:
s/data/params

Reply:
not sure what this means?

Reply:
it's a vim-ism for substitute the word "data" for "params", to be consistent with the terminology that feathers uses.

Reply:
gotcha! thanks